osquery/tables/system/linux/users.cpp

```diff
@@ -86,7 +89,23 @@ QueryData genUsers(QueryContext& context) {
 }
 
 QueryData genStartupItems(QueryContext& context) {
```

Comment:
Instead of adding it here, move this all to a new table. The only reason `genStartupItems` is here is because of https://github.com/facebook/osquery/pull/3137#issuecomment-292006780

Reply:
I would ask if this data is enumerated in other tables. If it is, there is now a macwin folder so we might be able to just remove all of this.